process/peer/process.go

```diff
@@ -1195,5 +1196,6 @@ func (vs *validatorStatistics) LastFinalizedRootHash() []byte {
 // EpochConfirmed is called whenever a new epoch is confirmed
 func (vs *validatorStatistics) EpochConfirmed(epoch uint32) {
 	vs.flagJailedEnabled.Toggle(epoch >= vs.jailedEnableEpoch)
+	vs.flagInactiveToJailCorrection.Toggle(epoch == vs.jailedEnableEpoch)
 	log.Debug("validatorStatistics: jailed", "enabled", vs.flagJailedEnabled.IsSet())
```

Comment:
Add also this log line: log.Debug("validatorStatistics: inactiveToJailCorrection", "enabled", vs.flagInactiveToJailCorrectionEnabled.IsSet())

Reply:
done